.gitignore

```diff
@@ -1 +1 @@
-.ipynb_checkpoints
+__pycache__
```

Comment:
You should just use https://github.com/github/gitignore/blob/main/Python.gitignore